app/app_delegate.rb

```diff
@@ -1,6 +1,6 @@
 # -*- encoding : utf-8 -*-
 class AppDelegate
-  def application(application, didFinishLaunchingWithOptions:launchOptions)
+  def application(_application, didFinishLaunchingWithOptions:launchOptions)
```

Comment:
Unused method argument - `didFinishLaunchingWithOptions`. You can also write as `application(*)` if you want the method to accept any arguments but don't care about them.